src/main/kotlin/view/OutputView.kt

```diff
@@ -28,11 +30,17 @@ object OutputView {
         var pathMarks = ""
 
         this.forEach { pathState ->
-            pathMarks += pathState.state
+            pathMarks += pathState.changeToMark()
         }
 
         return pathMarks
     }
 
+    private fun PathState.changeToMark(): String {
+        if (this == PathState.MOVE)
+            return MOVE_MARK
+        return STOP_MARK
+    }
```

Comment:
kotlin의 when식을 이용하면 이렇게도 작성할 수 있답니다 😊
```suggestion
    private fun PathState.changeToMark(): String = when (this) {
        PathState.MOVE -> MOVE_MARK
        PathState.STOP -> STOP_MARK
    }
```